profile/README.md

```diff
@@ -0,0 +1,26 @@
+# Bottlerocket
+
+Bottlerocket is a Linux-based operating system optimized for hosting containers.
+It’s free and open-source software, developed in the open here on GitHub.
+Bottlerocket is installed as the base operating system on the machine or instance where your containers themselves are running.
+It is specifically designed to work with your container orchestrator (like Kubernetes) to automate the lifecycle of the containers running in your cluster.
+Bottlerocket runs in the cloud or in your datacenter.
+
+## Project Navigation
+
+The `bottlerocket-os` GitHub organization contains a number of repos. Feel free to examine everything, but most people are looking for the following:
```

Comment:
nit: consider a slightly different order, from most generally relevant to most specific:
* bottlerocket
* bottlerocket-project-website (to mention bottlerocket.dev sooner)
* bottlerocket-control-container
* bottlerocket-admin-container
* bottlerocket-update-operator
* bottlerocket-ecs-updater

(It might also look better as a table with consistent widths for the project and description columns.)

Reply:
Your order seems fine. 

I'll play around with the tables. I have a bias against tables in Markdown since the syntax is pretty bad to use/edit/version control, but it might make sense in this context.